setup.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, *args, **kwargs):
     maintainer_email="info@tuxemon.org",
     url="https://www.tuxemon.org",
     include_package_data=True,
-    packages=modules,
+    packages=find_packages(),
```

Comment:
You may raise a PR for this change, but delete the code rather than commenting it out.